packages/blue-sdk/src/addresses.ts

```diff
@@ -492,11 +481,70 @@ entries(permissionedWrapperTokens).forEach(([chainId, tokens]) => {
  */
 export const convexWrapperTokens: Record<number, Set<Address>> = {
   [ChainId.EthMainnet]: new Set([
-    addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDTWBTCWETH-morpho"],
-    addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDCWBTCWETH-morpho"],
-    addressesRegistry[ChainId.EthMainnet]["stkcvxcrvCRVUSDTBTCWSTETH-morpho"],
-    addressesRegistry[ChainId.EthMainnet]["stkcvxTryLSD-morpho"],
-    addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDETHCRV-morpho"],
-    addressesRegistry[ChainId.EthMainnet]["stkcvx2BTC-f-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDTWBTCWETH-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDCWBTCWETH-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvxcrvCRVUSDTBTCWSTETH-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvxTryLSD-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDETHCRV-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvx2BTC-f-morpho"],
   ]),
 };
+
+export let addressesRegistry = deepFreeze(_addressesRegistry);
```

Comment:
Have you tried assigning a value to this variable outside the context of this file?
I believe it is not possible, but worth testing

Reply:
nope it's not possible as modules only expose getters